pkg/mcs/resourcemanager/server/metering.go

```diff
@@ -0,0 +1,127 @@
+// Copyright 2025 TiKV Project Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package server
+
+import (
+	"sync"
+
+	"github.com/pingcap/metering_sdk/common"
+
+	"github.com/tikv/pd/pkg/metering"
+)
+
+const (
+	resourceManagerCategory = "resource-manager"
+	ruMeteringVersion       = "1"
+	sourceName              = "pd"
+
+	meteringUnitRU    = "RU"
+	meteringUnitBytes = "Bytes"
+
+	meteringDataVersionField             = "version"
+	meteringDataClusterIDField           = "cluster_id"
+	meteringDataSourceNameField          = "source_name"
+	meteringDataOltpRUField              = "oltp_ru"
+	meteringDataOlapRUField              = "olap_ru"
+	meteringDataCrossAZTrafficBytesField = "cross_az_traffic_bytes"
+)
```

Comment:
Just a suggestion. Is it better to define them in `sdk` repo?

Reply:
Nice suggestion, have feedbacked to the SDK team.